### DISCUSSION TITLE
Intersection of NE 40th St with...itself (UDist/Wallingford)

### DISCUSSION THREAD

**Discussion by gussmith23** [dabreegster/abstreet#1089] [General]
Hi! I've known about this project for a while, as someone who's interested in civic hacking in Seattle. It's only over the past week that I realized I may have an application of A/B Street in the real world.

[This intersection](https://www.google.com/maps/@47.6557419,-122.3206423,18z) really stinks. I'm convinced it stinks for cars and pedestrians alike. I walk or bike through the intersection every day, and I can say that it's terrifying as a pedestrian to walk through a busy 5-way intersection that's only managed with stop signs. People are aggressive, perhaps because the intersection is a frustrating bottleneck. I've nearly been hit on foot or on bike multiple times in the past few years, by people who are trying to pay attention to the four other stopped cars (actually five, as one of the roads coming in is two lanes), and not paying attention to the pedestrian crosswalks. For cars, it's equally stressful (though less dangerous) as it's an utter free-for-all. People don't beep their horns in Seattle, but they do at this intersection!

(also, a hilarious fun fact: 4 of the 5 streets that make up this intersection are NE 40th Street. How does that make sense?? 😆)

I'm unsure how to apply A/B Street exactly in this case. I would potentially be interested in pushing this to local government, potentially with help from A/B Street simulations. Is there a usual way these things go?

**Comment by dabreegster** [comment]
Hi! I guess the first question is what you think could help at this intersection -- closing some of the roads to traffic or one-way? Converting to a full traffic signal (and if so, what should the stages/phases look like)? You can mock up those changes pretty easily. The next step of "measuring what changes" is less clear, but you can at least do things like watch people and vehicles move through the space. (Hover on the intersection in sandbox mode and press **z** to spawn agents)

I don't have ideas there, but nearby is https://www.openstreetmap.org/node/1752970097. While I lived in Seattle and cycled past here daily, I saw many near-misses from vehicles trying to enter or exit those two small roads on the hill. Long ago I made a simple edit in the simulation to just close the two roads to traffic and just show how the turning movements and conflicts are reduced:
![Screenshot from 2023-05-09 09-24-04](https://user-images.githubusercontent.com/1664407/237038561-4b32d98f-8029-4e41-81a2-02868a853314.png)
![Screenshot from 2023-05-09 09-23-36](https://user-images.githubusercontent.com/1664407/237038566-50f00dd7-11c0-4c36-8d51-6a411b546977.png)
I also wanted to demonstrate that those two roads aren't critical to the vehicle network and that the alternatives nearby are adequate and much safer/simpler crossings. I don't think I got very far, but running a simulation in the area and using the relative throughput layer to see what road vehicles take instead was helpful. I never wound up trying to take this to Neighborhood Greenways or the city as an idea though, so I don't know what evidence they'd like to see to support this sort of idea.
